src/test/java/org/snomed/snowstorm/core/data/services/ConceptServiceTest.java

```diff
@@ -1699,6 +1703,138 @@ private boolean waitUntil(Supplier<Boolean> supplier, int maxSecondsToWait) {
 		}
 	}
 
+	/**
+	 * This test illustrates a 'module jump' scenario.
+	 */
+	@Test
+	void testJumpModule() throws ServiceException {
+		// Setup
+		String mainBranchPath = "MAIN";
+		String extensionBranchPath = "MAIN/SNOMEDCT-SE";
+		String fsnTerm = "Bacon and Eggs (food)";
+		String synonymTerm = "English Breakfast";
+		String swedishFsnTerm = "Bacon och ägg";
+		String swedishLanguageReferenceSet = "551231000005103";
+		String parentConceptId = "107658001";
+		String anotherConceptId = "116680003";
+		String anotherDescriptionId = "218680004";
+		String extensionModule = "45991000052106";
+		String refSetId="123459876000000000";
+
+		Map<String, String> intPreferred = Map.of(US_EN_LANG_REFSET, descriptionAcceptabilityNames.get(PREFERRED),
+				GB_EN_LANG_REFSET, descriptionAcceptabilityNames.get(PREFERRED));
+		Map<String, String> svPreferred = Map.of(swedishLanguageReferenceSet, descriptionAcceptabilityNames.get(PREFERRED));
+		Map<String, String> svAcceptable = Map.of(swedishLanguageReferenceSet, descriptionAcceptabilityNames.get(ACCEPTABLE));
+		String ci = "CASE_INSENSITIVE";
+		conceptService.batchCreate(Lists.newArrayList(new Concept(parentConceptId), new Concept(anotherConceptId)), mainBranchPath);
```

Comment:
Does this just create a single IS_A ?

Add another inferred relationship as well please, so that we can inactivate that as well.    Some attribute - causative agent or something like that.